dluhc-component-library/src/components/timetablePage/TimetablePage.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from "preact/hooks";
+import Timetable from "./Timetable";
+
+interface TimetablePageProps {
+  filepath: string;
+}
+
+const loadData = async (filepath: string) =>
+  await fetch(filepath).then((res) => res.json());
+
+const TimetablePage = ({ filepath }: TimetablePageProps) => {
+  const [timetableData, setTimetableData] = useState();
```

Comment:
Could we pull `RowData` out into a new file so we can use it for the state type here too?

Reply:
Yeah ill do that when i pull the progress type out too